jdaviz/configs/specviz/tests/test_helper.py

```diff
@@ -248,12 +258,16 @@ def test_load_spectrum_list_directory(tmpdir, specviz_helper):
         sample_data_zip.extractall(tmpdir.strpath)
     data_path = str(tmpdir.join('NIRISS_for_parser_p0171'))
 
+    # Load two NIRISS x1d files from FITS. They have 19 and 20 EXTRACT1D
+    # extensions per file, for a total of 39 spectra to load:
     with pytest.warns(UserWarning, match='SRCTYPE is missing or UNKNOWN in JWST x1d loader'):
         specviz_helper.load_spectrum(data_path)
+
     # NOTE: the length was 3 before specutils 1.9 (https://github.com/astropy/specutils/pull/982)
     SPECUTILS_LT_1_9 = Version(specutils.__version__) < Version('1.9.0')
-    expected_len = 40 if not SPECUTILS_LT_1_9 else 3
+    expected_len = 39 if not SPECUTILS_LT_1_9 else 3
```

Comment:
This needs to be revised because we don't expect a concatenated spectrum to be loaded by default, so we expected the length [described here](https://github.com/spacetelescope/jdaviz/pull/1659/files#diff-5ef5cbec11fb8f2fb8f31547e9995778108c86e1f383e9676e2b9ad52830c1bfR291).

Reply:
If you're also bumping specutils here, I think you can safely get rid of the if statement entirely as part of this PR.

Reply:
Ok great, I wasn't sure if this had to do with backporting?

Reply:
If we are bumping specutils minversion, should we even backport?